src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -87,6 +87,44 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * To configure the check with two excluded classes {@code HashMap} and {@code HashSet}:
+ * </p>
+ * <pre>
+ * &lt;module name="ClassFanOutComplexity"&gt;
+ *   &lt;property name="excludedClasses" value="HashMap, HashSet"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * To configure the check with two regular expressions {@code ^Array.*} and {@code .*Exception$}:
```

Comment:
all configs should be followed by code block that contains violations and lines that are compliant.
It helps a lot tohave the same code (if reasonable) and change config and show how behavior is changing.
Please do.

Reply:
I've added the examples wherever I felt necessary. I think some configs were extremely obvious. Anyway, please have a look and let me know if something still needs to be fixed.
Thank you for your time!

Reply:
I do not see code example after default config

Reply:
I thought it was really trivial, self-explanatory and obvious that's why I skipped it. However, I've added it now.